common/src/stack/command/stack/commands/set/vm/storage/name/__init__.py

```diff
@@ -47,29 +47,20 @@ def run(self, params, args):
 
 			# A mountpoint won't be in the image list, so don't raise
 			# an error if it's not there
-			if backing not in disk_images and '/dev' not in backing:
+			if backing not in disk_images and '/dev/' not in backing:
```

Comment:
Shouldn't this be using startswith()?

```
if backing not in disk_images and not backing.startswith('/dev/'):
```

Or is /mnt/someplace/dev/sde/ a reasonable "backing"?

Reply:
It isn't, and mountpoints can only start with `/dev/` in their path.